pkgs/os-specific/linux/esdm_es/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, kernel, kmod, esdm }:
+
+stdenv.mkDerivation rec {
+  pname = "esdm_es";
+  inherit (esdm) version;
+
+  inherit (esdm) src;
+
+  sourceRoot = "source/addon/linux_esdm_es";
+
+  hardeningDisable = [ "format" "pic" ];
```

Comment:
Hardening disable requires a comment to give a rationale on why this is needed.

Reply:
You have a point there. It also works without this line. It is now removed.